### PR TITLE
fix: only toggle the navbar button if it's visible

### LINKED ISSUE #61: Navigation bar blinking upon click
It seems that the top navigation bar started to blink/pulsate in a strange way upon click.

https://user-images.githubusercontent.com/3430939/113915190-c0510080-97de-11eb-868e-b3dac6e1d540.mp4

It is unclear whether it is a side effect from https://github.com/falconry/falconry.github.io/pull/56 (I think not, but not 100% sure), or a new effect caused by bumping dependency versions.

### PR DESCRIPTION
**Pull request by CaselIT**
closes #61

### CODE REVIEW

File: src/dynamics.js [hunk 1]
```diff
@@ -111,7 +111,9 @@ function handleClick() {
     return;
   }
   document.getElementById('navbarPrimary').addEventListener('click', function(event) {
-    button.click();
+    if(getComputedStyle(button).display !== 'none'){
```
Comment (vytas7) [inline src/dynamics.js:114]: Not sure about the indentation and whitespace standards in this project, if any.

Shouldn't we add a space after `if`, and before `{`?

**Review by kgriffs — APPROVED** [review]
Tested it and works for me in Safari.

**Review by vytas7 — APPROVED** [review]
LGTM bar the code formatting question.

File: src/dynamics.js (hunk 1 above)
Comment (CaselIT) [reply src/dynamics.js:114]: we could add prettier or something, but I think it could be another issue

Comment (vytas7) [reply src/dynamics.js:114]: Good enough for me. I just noticed the surrounding code does have spaces in these places.

Comment (CaselIT) [reply src/dynamics.js:114]: I use the builtin formatter of vscode and for js/ts does just the bare minimum, so it will not add/remove the spaced there.

Comment (vytas7) [reply src/dynamics.js:114]: Ah, fair enough.

Comment (CaselIT) [reply src/dynamics.js:114]: opened https://github.com/falconry/falconry.github.io/issues/63 just as a reminder if we want to go that way

**Pull request merged**